Libretro build failure on Linux x86_64

As of 2f511530bdddec3a282008bec84f1d3828ac81b3, sorry it took so long to report this time:
```
../Core/GbDebugger.o: In function `GbDebugger::GbDebugger(Debugger*)':
GbDebugger.cpp:(.text+0x582): undefined reference to `GbEventManager::GbEventManager(Debugger*, GbCpu*, GbPpu*)'
../Core/GbDebugger.o: In function `GbDebugger::ProcessRead(unsigned short, unsigned char, MemoryOperationType)':
GbDebugger.cpp:(.text+0xe04): undefined reference to `GbEventManager::AddEvent(DebugEventType, MemoryOperationInfo&, int)'
../Core/GbDebugger.o: In function `GbDebugger::ProcessWrite(unsigned short, unsigned char, MemoryOperationType)':
GbDebugger.cpp:(.text+0x11e7): undefined reference to `GbEventManager::AddEvent(DebugEventType, MemoryOperationInfo&, int)'
../Core/GbDebugger.o: In function `GbDebugger::ProcessInterrupt(unsigned int, unsigned int)':
GbDebugger.cpp:(.text+0x1290): undefined reference to `GbEventManager::AddEvent(DebugEventType)'
../Core/GbDebugger.o: In function `std::_Sp_counted_ptr<GbEventManager*, (__gnu_cxx::_Lock_policy)2>::_M_dispose()':
GbDebugger.cpp:(.text._ZNSt15_Sp_counted_ptrIP14GbEventManagerLN9__gnu_cxx12_Lock_policyE2EE10_M_disposeEv[_ZNSt15_Sp_counted_ptrIP14GbEventManagerLN9__gnu_cxx12_Lock_policyE2EE10_M_disposeEv]+0xe): undefined reference to `GbEventManager::~GbEventManager()'
../Core/GbDebugger.o: In function `GbDebugger::GbDebugger(Debugger*) [clone .cold]':
GbDebugger.cpp:(.text.unlikely+0x13b): undefined reference to `GbEventManager::~GbEventManager()'
collect2: error: ld returned 1 exit status
Makefile:314: recipe for target 'mesen-s_libretro.so' failed
```
GCC Version 9.3.0-11

--- a/Libretro/Makefile.common
+++ b/Libretro/Makefile.common
@@ -69,6 +69,7 @@ SOURCES_CXX := $(LIBRETRO_DIR)/libretro.cpp \
                $(CORE_DIR)/GbTimer.cpp \
                $(CORE_DIR)/GbMemoryManager.cpp \
                $(CORE_DIR)/GbDebugger.cpp \
+               $(CORE_DIR)/GbEventManager.cpp \
                $(CORE_DIR)/GameboyDisUtils.cpp \
                $(CORE_DIR)/Gsu.cpp \
                $(CORE_DIR)/Gsu.Instructions.cpp \

